### ISSUE TITLE
Efficiency vs security in dynamic considerations

### ISSUE THREAD

**Issue by trishankatdatadog** [uptane/deployment-considerations#34]
This [text](https://github.com/uptane/deployment-considerations/blob/59f8b1adc4f30ee4a3cc2620c527aee8b4c25803/customizations.md#picking-an-option-efficiency-vs-security) as it is is not as clear as it could be.

It alleges efficiency considerations when including dynamic directions in the custom targets metadata: "If dynamic directions are sent using the custom field of Targets metadata, these directions will be downloaded by all ECUs on the car."  But this is not clear. I think it's only true when the targets metadata is from the Director repo, not the Image repo.

In short, this text needs some clarifications.

**Comment by jhdalek55** [comment]
@trishankatdatadog , when you get a chance, can you rough out a suggested edit for this issue? Or perhaps @mnm678, you could take a stab at this, as I believe you worked on this issue in the Google doc?

**Comment by trishankatdatadog** [comment]
I will do my best, drowning with work

**Comment by jhdalek55** [comment]
Understood. No pressure here.

**Comment by jhdalek55** [comment]
@trishankatdatadog @mnm678 ...No rush on this, but if this is a simple text clarification, can one of you take a stab at drafting this? It would be good to have this in place before we begin work on doing a stable PDF of the pages. Thanks.

**Comment by trishankatdatadog** [comment]
Done, I added #43 as a fix
